### DISCUSSION TITLE
"AUC/Student history percentage" plots

### DISCUSSION THREAD

**Discussion by clabra** [pykt-team/pykt-toolkit#106] [Q&A]
Hi, 

I'm trying to plot "AUC/Student history percentage" curves from results obtained after training some models with pykt-toolkit. Is there any tool included in your github repo to do so?. If not, where can I find the results' data in order to plot this kind of curve?. 

Thanks and congrats for the project.

**Comment by sonyawong** [comment]
> Hi,
> 
> I'm trying to plot "AUC/Student history percentage" curves from results obtained after training some models with pykt-toolkit. Is there any tool included in your github repo to do so?. If not, where can I find the results' data in order to plot this kind of curve?.
> 
> Thanks and congrats for the project.

- Hi, the corresponding codes of experiment results in visualization does not provide in the current repo. You can plot the curve with the results yourself if you need.  On the other hand, we will provide the codes and other benchmark results in about two weeks. Thx.
